index.html

```diff
@@ -0,0 +1,180 @@
+            </ul>
+            <ul class="Redux_React">
+                <li><img src="images/lecture-icon.png" class="icon" alt="lecture icon"></li>
+                <li class="RR_course">Redux and React</li>
+                <li class="RR_des">Redux and React are two major areas that any app developer should put emphasis on.</li>
+            </ul>
+            <ul class="Andriod_N_Developer">
+                <li><img src="images/lecture-icon.png" class="icon" alt="lecture icon"></li>
+                <li class="AND_course">Andriod N Developer</li>
+                <li class="AND_des">Are you someone who loves to see the new apps popping up in Google Play feeds?</li>
+            </ul>
+            <ul class="Cyber_security">
+                <li><img src="images/lecture-icon.png" class="icon" alt="lecture icon"></li>
+                <li class="CS_course">Cyber Security</li>
+                <li class="CS_des">You can become an expert in Cybersecurity in different operating systems like macOS, Linus, iOS.</li>
+            </ul>
+            <div class="Btn">
+                <button class="course_btn">Join Orbit Innovations</button>
+            </div>
+        </div>
+        <div class="see">
+            <a class="see_program" href="#">VIEW OUR LECTURERS</a>
+        </div>
+    </main>
+    <section id="lecturers"  class="featured">
+        <div class="feat-heading">
+            <h2 class="featured_lecturers">Featured Lecturers</h2>
+            <hr id="hr-feature">
+        </div>
+        <ul class="course_lecturers" id="lecturer-list"></ul>
+        <div class="dropdown">
+            <button class="more_lecturers">MORE<img src="images/arrow-down-icon.png" class="down"></button>
+
+        </div> 
+    </section>
+    <div class="hidden_features">
+        <section id="partners" class="partners">
+            <div class="feat-heading">
+                <h2 class="partner_home">Partners</h2>
+                <hr id="hr-partner">
+             </div>
+            <div class="desktop_partners">
+                <ul>
+                    <li><img src="images/dcs-removebg.png" class="durham_college" alt="logo of Durham College"><img src="images/trilogy-removebg.png" class="trilogy_innovation" alt="logo of trilogy innovation"><img src="images/power-tech-removebg.png" class="power_technology" alt="logo of Power Technology School"><img src="images/bcc-removebg.png" class="blue_crest" alt="logo of Blue Crest"></li>
+                </ul>
+            </div>
+        </section>
+        <section class="footer_home">
+            <div class="footer_wrapper">
+                <ul class="footer_info">
+                    <li class="copyright_home">&copyOrbit<br>Web Innovations</li>
+                    <li class="reserved_home"> 2020 Orbit Innovations Web. All Rights Reserved</li>
+                </ul>
+            </div>
+        </section>
```

Comment:
- [ ] I think it will be semantically more correct if you use  a `footer` element for this section. 🥸️